test/formatters/currency.js

```diff
@@ -46,6 +46,15 @@ test("formats cents", t => {
   });
 });
 
+test("Properly handles negitive amounts", t => {
```

Comment:
Formatter was not stripping the "-" but i wanted to add a test to make sure we do not break it in the future.